Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,1249 @@
+    account_ids: List[str] = []
+
+    if len(res) == 1:
+        account_ids = [res[0].get('accountId', '')] if is_jira_cloud else [res[0].get('name', '')]
+
+    elif(is_jira_cloud):
+        # We check the displayName and emailAddress, because that is what the Cloud API returns
+        account_ids = [
+            user.get('accountId', '') for user in res if (attribute.lower() in [user.get('displayName', '').lower(),
+                                                                                user.get('emailAddress', '').lower()])
+        ]
+
+    else:
+        # We check the displayName, emailAddress, and name, because that is what the OnPrem API returns
+        account_ids = [
+            user.get('name', '') for user in res if (attribute.lower() in [user.get('displayName', '').lower(),
+                                                                           user.get('emailAddress', '').lower(),
+                                                                           user.get('name', '').lower()])
+        ]
+    if not account_ids:
+        # The email address is a private account field and sometimes is blank, therefore, if the response is not empty but
+        # account_ids is, the user should try "DisplayName" attribute.
+        demisto.debug(f'Multiple accounts found, but it was not possible to resolve which one of them is most '
+                      f'relevant to attribute {attribute}.')
+        return CommandResults(readable_output=(f'Multiple accounts found, but it was not possible to resolve which one'
+                                               f' of them is most relevant to attribute {attribute}. Please try to provide'
+                                               ' the "DisplayName" attribute if not done so before, or supply the full attribute.'))
+
+    elif len(account_ids) > 1:
+        return CommandResults(readable_output=f'Multiple account IDs were found for attribute: {attribute}.\n'
+                              f'Please try to provide the other attributes available - Email or DisplayName'
+                              ' (and Name in the case of Jira OnPrem).')
+    # If reached here, that means there is only one entry in account_ids that holds the right id for the given attribute
+    outputs['AccountId'] = account_ids[0]
+    return CommandResults(
+        outputs_prefix='Jira.User',
+        outputs_key_field='AccountId',
+        outputs=outputs,
+        readable_output=f'The account ID that holds the attribute `{attribute}`: {outputs["AccountId"]}'
+    )
+
+    # Board Commands
+
+    # ...
+
+    # Fetch
+
+    # Polling
+
+
+def ouath_start_command(client: JiraBaseClient, args: Dict[str, Any] = None) -> CommandResults:
+    url = client.oauth_start()
+    return CommandResults(readable_output=('In order to retrieve the authorization code, please authorize'
+                                           f' yourself using the following link:\n{create_clickable_url(url)}'))
```

Comment:
If this is to authorize using the cloud please expain they need to take the code from the html